cli/test/fixtures/static-server.js

```diff
@@ -188,9 +191,8 @@ class Server {
         <h1>Smoke test fixtures</h1>
         ${fixturePaths.map(p => `<a href=${encodeURI(p)}>${escape(p)}</a>`).join('<br>')}
       `;
-      response.writeHead(200, {
-        'Content-Security-Policy': `default-src 'none';`,
-      });
+      response.statusCode = 200;
+      response.setHeader('Content-Security-Policy', `default-src 'none';`);
```

Comment:
I'm can't find any info in the docs, but for some reason `response.writeHead` will end the response in Node 20 and send it to the user. We want to be able to amend the headers and status code later in execution (e.g. if there was a redirect) so we should use `setHeader` and `statusCode =` now.